src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -178,6 +178,40 @@ protected boolean isFinished() {
         };
     }
 
+    public Command ManualLift() {
+        return new SubsystemCommand(this) {
+            @Override
+            protected void initialize() {
+                disengageBrake();
+                disable();
+                liftDrive.set(0);
+            }
+
+            @Override
+            protected void execute() {
+                if (topLimitSwitch.get() == true || bottomLimitSwitch.get() == true) {
+                    liftDrive.set(0);
+                } else
+                    enable();
```

Comment:
Should this be setting `liftDrive` to something instead?

Reply:
this should be `liftDrive.set(0)`, not enable. Also remove the `liftDrive.set(0);` line in the initialize. 

We will eventually change the 0 to getting a value from a joystick.